centml/templates/notebook-deployment.yaml

```diff
@@ -49,46 +49,13 @@ spec:
               memory: 8Gi
             limits:
               # GPUs can only be allocated as a limit, which both reserves and limits the number of GPUs the Pod will have access to
-              # Making individual Pods resource light is advantageous for bin-packing. In the case of Jupyter, we stick to two GPUs for
+              # Making individual Pods resource light is advantageous for bin-packing. In the case of Jupyter, we stick to one GPU for
               # demonstration purposes
               nvidia.com/gpu: 1
+              cpu: "4"
+              memory: 18Gi
```

Comment:
The CPU limits should not be lower than the requests. Also please set the mem requests and limits equal.

Reply:
Done!